bot.py

```diff
@@ -51,11 +51,11 @@ def check(update, context):
 def main():
     updater = Updater(token, use_context=True)
     dp = updater.dispatcher
-    dp.add_handler(CommandHandler("help", help))
-    dp.add_handler(CommandHandler("xkcd", xkcd))
-    dp.add_handler(CommandHandler("links", links))
-    dp.add_handler(CommandHandler("set_meetup", set_meetup))
-    dp.add_handler(CommandHandler("meetup", meetup))
+    dp.add_handler(MessageHandler(Filters.regex(r'^(/help@%s)$' % bot_name), help))
+    dp.add_handler(MessageHandler(Filters.regex(r'^(/xkcd@%s)$' % bot_name), xkcd))
+    dp.add_handler(MessageHandler(Filters.regex(r'^(/links@%s)$' % bot_name), links))
+    dp.add_handler(MessageHandler(Filters.regex(r'^(/set_meetup@%s)$' % bot_name), set_meetup))
+    dp.add_handler(MessageHandler(Filters.regex(r'^(/meetup@%s)$' % bot_name), meetup))
```

Comment:
Do we need to change the handlers from `CommandHandler` to `MessageHandler`? :thinking: 
Ig we can use `Filters` with `CommandHandler` too.

Reply:
Yeah, I have tried using `CommandHandler ` but it's giving "regex is not iterable" error. I think `MessageHandler` will be fine for now.